bundles/org.openhab.binding.sensorpush/README.md

```diff
@@ -0,0 +1,74 @@
+
+### cloudbridge thing
+
+The `cloudbridge` thing is responsible for communicating with the SensorPush Gateway Cloud.
+You must supply your user name and password.
+The poll and timeout parameters are optional.
+
+Parameters:
+
+* `user` (required) Your SensorPush cloud service user name (email address)
+* `password` (required) Your SensorPush cloud service password
+* `poll` (2-60, default = 5) Polling interval in minutes
+* `timeout` (1-120, default = 30) Timeout period for API requests in seconds.
+
+**Note:** To activate your API access, you must log in to the [Gateway Cloud Dashboard](https://dashboard.sensorpush.com/) and agree to the terms of service.
+Once you've logged in that initial time, your account will have access.
+
+### sensor thing
+
+The `sensor` thing represents an individual SensorPush sensor.
+It has a variety of channels that will be populated with the latest sensor readings at each poll period.
+Sensors relay their readings at approximate 1 minute intervals, so in normal operation the oldest a reading should be is approximately 1 minute plus the configured poll interval.
+The `time` channel will contain the timestamp of the latest readings.
+If your particular sensor model does not support a given channel, the value for that channel will remain NULL.
+The id parameter must be supplied.
+
+Parameters:
+
+* `id` (required) The unique ID number of the sensor
+* `pressureMode` The reporting mode for barometric pressure. Must be set to either "station" or "meteorological". The station mode reports the pressure as recorded by the sensor, while the meteorological mode adjusts the reported pressure to mean sea level as is common for weather reporting. Defaults to "station".
+* `altitude` The altitude of the sensor in feet above MSL. It is only necessary to set this parameter if you selected the "meteorological" option for pressureMode and have not set the sensor altitude in the SensorPush app. The altitude set in the SensorPush app will override this value.
+
+## Channels
+
+The following channels are provided by the `sensor` thing:
+
+| channel     | type                     | description              |
+|-------------|--------------------------|--------------------------|
+| temperature | Number:Temperature       | Temperature              |
+| humidity    | Number:Dimensionless     | Relative humidity        |
+| pressure    | Number:Pressure          | Barometric pressure      |
+| dewpoint    | Number:Temperature       | Dew point                |
+| vpd         | Number:Pressure          | Vapor pressure deficit   |
+| time        | DateTime                 | Time of reading          |
+| rssi        | Number:Power             | Received signal strength |
+| voltage     | Number:ElectricPotential | Battery voltage          |
+
+Note that all channels except `time` use QuantityType values.
+
+No channels are provided by the `cloudbridge` thing.
```

Comment:
I purposely left this out because I don't recommend using a text configuration for this binding. Auto-discovery is the way to go, since there is no easy way to determine the sensor IDs.

Reply:
There are many users who use textual configuration. What do you mean exactly by "no easy way"?

Reply:
The sensor IDs aren't exposed to the user at all, unless they write code to query the API. So auto-discovery is required.

Reply:
I've added a config file example for the bridge.